build_web_compilers/lib/src/dev_compiler_builder.dart

```diff
@@ -39,13 +39,21 @@ class DevCompilerBuilder implements Builder {
   /// directory, which contains the platform kernel files.
   final String platformSdk;
 
+  /// The absolute path to the libraries file for the current platform.
+  ///
+  /// If not provided, defaults to "lib/libraries.json" in the sdk directory.
+  final String librariesPath;
```

Comment:
Does this need to be absolute? Is it not relative to platformSdk?

Reply:
Nvm I guess its absolute in kernel builder so absolute here is fine too for consistency.